### PR TITLE
added normalization filter

### PR DESCRIPTION
**Pull request by jspaaks**
Refs #82

### CODE REVIEW

File: matchms/filtering/normalize_intensities.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def normalize_intensities(spectrum):
+    """Normalize intensities to unit height."""
+    scale_factor = max(spectrum.intensities)
```
Comment (fdiblen) [inline matchms/filtering/normalize_intensities.py]: I think we could use numpy functions for max and for normalization but not very important at this point.

File: tests/test_normalize_intensities.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+def test_normalize_intensities():
+
+    mz = numpy.array([10, 20, 30, 40], dtype='float')
+    intensities = numpy.array([0, 1, 10, 100], dtype='float')
+    spectrum = Spectrum(mz=mz, intensities=intensities, metadata=dict())
+
+    normalize_intensities(spectrum)
+
+    assert max(spectrum.intensities) == 1.0, "Expected the spectrum to be scaled to 1.0."
+    assert numpy.array_equal(spectrum.mz, mz), "Expected the spectrum's intensity locations along the mz-axis to be unaffected."
```
Comment (fdiblen) [inline tests/test_normalize_intensities.py]: so this ignores intensity values?

Comment (jspaaks) [reply tests/test_normalize_intensities.py]: Not sure I fully understand the question, but ``normalize_intensities`` is supposed to only change the values in ``spectrum.intensities``, specifically, it scales the full array such that its maximum is 1.0.

File: matchms/filtering/normalize_intensities.py (hunk 1 above)
Comment (jspaaks) [reply matchms/filtering/normalize_intensities.py]: updated.

**Review by fdiblen — APPROVED** [review]

**Comment by jspaaks** [comment]
thanks for reviewing!

**Pull request merged**